packages/web-pkg/src/composables/capability/useCapability.ts

```diff
@@ -81,3 +81,4 @@ export const useCapabilityFilesSharingPublicAlias = createCapabilityComposable(
   'files_sharing.public.alias',
   false
 )
+export const useCapabilityNotifications = createCapabilityComposable('notifications', {})
```

Comment:
You can also chain them like done above, e.g. `notifications.ocs-endpoints`. Then you can provide `[]` as default, which make it a bit easier to use.